backend/spec/requests/api/v1/enums_spec.rb

```diff
@@ -1,12 +1,22 @@
-require "rails_helper"
+require "swagger_helper"
 
 RSpec.describe "API V1 Enums", type: :request do
-  describe "GET #index" do
-    it "should return enums" do
-      get "/api/v1/enums"
+  path "/api/v1/enums" do
+    get "Returns list of the enums" do
+      tags "Enums"
+      produces "application/json"
 
-      expect(response).to have_http_status(:ok)
-      expect(response.body).to match_snapshot("api/v1/enums")
+      response "200", :success do
+        schema type: :object, properties: {
+          data: {type: :array, items: {"$ref" => "#/components/schemas/enum"}}
+        }
+
+        run_test!
+
+        it "matches snapshot", generate_swagger_example: true do
```

Comment:
to reuse examples generated by factories, I have added flag `generate_swagger_example ` -- it identifies test which response should be used as example at documentation